exporter/azure_monitor/example/ocServer.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"go.opencensus.io/plugin/ochttp"
+)
+
+func main() {
+	originalHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte("Hello, World!"))
+	})
+	och := &ochttp.Handler{
+		Handler: originalHandler, // The handler you'd have used originally
+	}
+
+	// Now use the instrumnted handler

```

Comment:
typo, might be good to report it.

Reply:
How did I not see that? Yes, I will report that later today. I'll give you credit.